spec/javascripts/app/views/help_view_spec.js

```diff
@@ -87,4 +87,51 @@ describe("app.views.Help", function(){
       expect(this.view.$el.find('#faq').children().first().hasClass('faq_question_miscellaneous')).toBeTruthy();
     });
   });
+
+  describe("findSection", function() {
+    beforeEach(function() {
+      this.view.render();
+    });
+
+    it('should return null for an unknown section', function() {
+      expect(this.view.findSection('you_shall_not_pass')).toBeNull();
+    });
+
+    it('should return the correct section link for existing sections', function() {
+      var sections = [
+        'account_and_data_management',
+        'aspects',
+        'pods',
+        'keyboard_shortcuts',
+        'tags',
+        'miscellaneous'
+      ];
+
+      var self = this;
+      _.each(sections, function(section) {
+        var el = self.view.$el.find('a[data-section=' + section + ']');
+        expect(self.view.findSection(section).html()).toBe(el.html());
+      });
+    });
+  });
+
+  describe("menuClicked", function() {
+    beforeEach(function() {
+      this.view.render();
+    });
+
+    it('should rewrite the location', function(){
+      var sections = [
+        'account_and_data_management',
+        'miscellaneous'
+      ];
+      spyOn(app.router, 'navigate');
```

Comment:
Thanks! Already did this 2 hours ago in the [next commit](https://github.com/svbergerem/diaspora/commit/c5468f072fb24e4185a8800d11af3206ee8b1d05). ;-)